infra/pipelines/docker/Dockerfile

```diff
@@ -1,10 +1,18 @@
-FROM prefecthq/prefect
+FROM graphistry/graphistry-blazing:v2.29.3
 
-RUN pip install supervisor arrow graphistry pyarrow simplejson twarc neo4j
-RUN prefect agent install local > supervisord.conf
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends supervisor \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN source activate rapids \
+    && pip install prefect==0.10.1 simplejson twarc neo4j boto3==1.12.39 \
+    && ( prefect agent install local > supervisord.conf )
 
 COPY . .
-RUN prefect backend server
 
-RUN date > /dev/null
+#TODO find cleaner way to avoid talking to cloud server
```

Comment:
Not sure what you mean - the agent needs to query the server, right? Also - just to be clear, this line doesn't talk to the server AFAIK - it switches the backend to "server" so the agent does not query Prefect Cloud.